First objective carsus atomic data

spectre900 opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->

## Description
<!--- Describe your changes in detail. -->

This PR is for the first objective of the Carsus Atomic Data project, mentored by Andreas Flörs, Christian Vogl and Ezequiel Passaro. 
It reads the tabulated data conained in the file si2_osc_kurucz, downloaded from [http://kookaburra.phyast.pitt.edu/hillier/web/CMFGEN.htm](http://kookaburra.phyast.pitt.edu/hillier/web/CMFGEN.htm)

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
First objective of the Carsus Atomic Data project GSoC 2021.

## How Has This Been Tested?
- [ ] Testing pipeline
- [ ] Reference Data Comparison following these [instructions](https://tardis-sn.github.io/tardis/development/continuous_integration.html)
- [ ] Other (please describe)
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, tests ran to see how -->
<!--- your change affects other areas of the code, etc. -->

## Screenshots (if appropriate):
Table - 1
![Screenshot from 2021-03-18 22-25-26](https://user-images.githubusercontent.com/31584170/111666140-73aa8300-8839-11eb-9403-3101ca2eab02.png)

Table - 2
![Screenshot from 2021-03-18 22-25-29](https://user-images.githubusercontent.com/31584170/111666151-76a57380-8839-11eb-9b22-6c69141f36ee.png)



## Types of changes
<!--- Go over all the following points, and put an `x` in all the boxes that apply: -->
<!--- What types of changes does your code introduce? -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] None of the above (please describe)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply: -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have built the documentation on my fork following [these instructions](https://tardis-sn.github.io/tardis/development/documentation_preview.html)
- [ ] I have assigned and requested two reviewers for this pull request

epassaro commented:
Hi @spectre900 , sorry for the delay. It looks good. 

Try to split the transition column.

spectre900 commented:
Hi @epassaro ,

I have updated the code as per your review
Also attached the updated screenshot here

Updated Table 2 -
![Screenshot from 2021-03-23 10-17-34](https://user-images.githubusercontent.com/31584170/112095269-3429c100-8bc2-11eb-98b3-58605a199ddf.png)
